github/internals/gql/queries.py

```diff
@@ -0,0 +1,79 @@
+"""GitHub GraphQL queries module"""
+from typing import Dict, Text
+
+def make_pull_requests_query(owner: Text, repo: Text) -> Dict[Text, Text]:
+    return {"query": """{
+  repository(owner:"%s", name:"%s") {
+    url
+    pullRequests(last: 50, states: OPEN) {
+      nodes {
+        number
+        baseRefName
+        mergeable
+        author {
+          login
+        }
+        labels(last: 5) {
+          nodes {
+            name
+          }
+        }
+        commits(last: 1) {
+          nodes {
+            commit {
+              oid
+              status {
+                contexts {
+                  context
+                  description
+                  state
+                  targetUrl
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+  rateLimit {
+    limit
+    cost
+    remaining
+    resetAt
+  }
+}""" % (owner, repo)}
```

Comment:
Please use `str.format()` instead.

Reply:
But it would not work in this case...
```python
>>> "{ bla { blabla { {bar} } } }".format(bar="bar")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: unexpected '{' in field name
```
And to make it work any curly braces which are not the part of formatting placeholder:
```python
>>> "{{ bla {{ blabla {{ {bar} }} }} }}".format(bar="bar")
'{ bla { blabla { bar } } }'
```

I really don't mind to change it if you're ok with doubling :)